src/screens/OperationDetails.js

```diff
@@ -0,0 +1,175 @@
+/* @flow */
+import React, { Component } from "react";
+import moment from "moment";
+import {
+  View,
+  StyleSheet,
+  ScrollView,
+  Linking,
+  Image,
+  TouchableOpacity
+} from "react-native";
+import type { NavigationScreenProp } from "react-navigation";
+import HeaderRightClose from "../components/HeaderRightClose";
+import LText from "../components/LText";
+import CurrencyIcon from "../components/CurrencyIcon";
+import SectionEntry from "../components/SectionEntry";
+import BlueButton from "../components/BlueButton";
+import CurrencyUnitValue from "../components/CurrencyUnitValue";
+import CounterValue from "../components/CounterValue";
+import colors from "../colors";
+
+export default class OperationDetails extends Component<{
+  navigation: NavigationScreenProp<*>
+}> {
+  static navigationOptions = ({ navigation }: *) => ({
+    title: "Operation Details",
+    headerRight: <HeaderRightClose navigation={navigation} />,
+    headerLeft: null
+  });
+
+  viewOperation = () => {
+    const { operation } = this.props.navigation.state.params;
+    Linking.openURL(`https://testnet.blockchain.info/tx/${operation.id}`).catch(
```

Comment:
note: we need to think how to generalize that.. probably will come from settings, but it will be "settings per crypto currency"